_data/software/vagrant.yml

```diff
@@ -0,0 +1,37 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      Vagrant is an open-source software product for building and maintaining
+      portable virtual software development environments.
+    fr: >-
+      Vagrant est un produit logiciel open source pour la création et la
+      maintenance d'environnements de développement de logiciels virtuels
+      portables.
+category: '43233000'
+homepageURL:
+  en: 'https://www.vagrantup.com/'
+  fr: 'https://www.vagrantup.com/'
+licences:
+  - URL:
+      en: 'https://github.com/hashicorp/vagrant/blob/master/LICENSE'
+      fr: 'https://github.com/hashicorp/vagrant/blob/master/LICENSE'
+    spdxID: MIT
+name:
+  en: Vagrant
+  fr: Vagrant
+tags:
+  en:
+    - VM
```

Comment:
As a later thing, for the repos that use the GitHub tagging like https://github.com/hashicorp/vagrant we could see which ones make sense to bring over